applications/luci-app-udpxy/htdocs/luci-static/resources/view/udpxy.js

```diff
@@ -27,18 +28,24 @@ return view.extend({
 
 		o = s.option(form.Flag, 'status', _('Client statistics'));
 
-		o = s.option(form.Value, 'bind', _('HTTP Listen interface'));
-		o.datatype = 'or(ipaddr, network)';
-		o.placeholder = '0.0.0.0 || lan1';
+		o = s.option(widgets.DeviceSelect, 'bind', _('HTTP Listen interface'));
```

Comment:
It's clear that what you did is more than "add source_network option".
Using `widgets.DeviceSelect` here can be misleading as it's only for interface. Though you can put a ipaddr you can see a confusing prefix "custom interface:".